README.md

```diff
@@ -148,6 +198,54 @@ class CdkStack(SceptreCdkStack):
     # and then you can add your resources...
 ```
 
+### Importing from other modules
```

Comment:
This section I just moved up from lower in the Readme. It seemed more appropriately placed after the section on creating your stack class.